app/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Article;
+use Illuminate\Http\Request;
+
+class ArticleController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $articles = \App\Article::all();
```

Comment:
You have `use App\Article;` at the beginning of your file so you can shorten this statement to `$articles = Article::all();`